<launch>
    <arg name="agent" default="ddpg"/>
    <arg name="model" default="actor_critic_1"/>
    <rosparam command="load" file="$(find rl_agents)/cfg/agent.yaml" />

    <!-- launch rl_agent internal launch file -->
    <include file="$(find rl_agents)/launch/$(arg agent).launch">
        <arg name="model" value="$(arg model)"/>
    </include>
</launch>